src/style.css

```diff
@@ -1,3 +1,68 @@
 body {
   margin: 0;
+
+  --selected-color: #00acdc;
+}
+
+.header {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  min-width: 1024px;
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.header__logo__img {
+  vertical-align: bottom;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+
+  list-style: none;
+}
+
+.nav__list > li:first-child a {
```

Comment:
Simplify selectors to adhere to BEM principles and avoid increasing specificity unnecessarily. We can simplify this by using the fact that nav__link have also is-active class.